.github/workflows/lint.yml

```diff
@@ -16,5 +16,5 @@ jobs:
           node-version: 16.x
           cache: npm
       - run: yarn install --frozen-lockfile
-      - run: yarn run lint
+      - run: yarn run lint --disable MD013
```

Comment:
why are we adding this line?

Reply:
i can remove it, but similarly to Mustafa asking why we have linting on rollkit docs in slack a bit ago

Reply:
i am removing this.